iib_modules/fac_os.c

```diff
@@ -187,10 +183,10 @@ void check_fac_os_indication_leds()
 
 /////////////////////////////////////////////////////////////////////////////////////////////
 
-    //Fuga para o Terra
-    if(fac_os.GroundLeakageItlkSts) Led7TurnOff();
-    else if(fac_os.GroundLeakageAlarmSts) Led7Toggle();
-    else Led7TurnOn();
+    //Reservado
```

Comment:
In case we reserve one LED for future use, I suggest to move it to the last one (Led10). Otherwise, I suggest to divide the two indications from Led6 like this:

Led6: Heat-sink overtemperature
Led7: Inductor overtemperature

Reply:
It was discussed with the CoP group and the condition was reached that LED7 would be reserved for a future implementation of voltage reading in the FAC-OS module